### PR TITLE
Remove deprecated packages and update test logic

### PR DESCRIPTION
**Pull request by brwnrclse**
Tookout some packages I missed before, and updated the test code accordingly. Also bumped the version number so there can be a new release on npm. 🤞

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
 .DS_Store
-node_modules
\ No newline at end of file
+node_modules
+.nyc_output
```
Comment (brwnrclse) [inline .gitignore]: New generated files from nyc

File: package.json [hunk 2]
```diff
@@ -1,29 +1,28 @@
...
+    "rimraf": "^2.6.3",
     "stylus": "0.54.5"
   },
   "author": "Jeff Escalante",
   "main": "index.js",
   "scripts": {
-    "test": "mocha",
-    "coveralls": "istanbul cover _mocha --report lcovonly -- -R spec && cat ./coverage/lcov.info | coveralls && rm -rf ./coverage",
-    "coverage": "istanbul cover _mocha --report html -- -R spec && open coverage/index.html"
+    "test": "nyc mocha",
```
Comment (brwnrclse) [inline package.json]: https://istanbul.js.org/

**Comment by jescalan** [comment]
I'll run the version bump and release after this is merged, no need to change the version number!

**Comment by jescalan** [comment]
Closes by #181

**Pull request closed without merging**